src/index.js

```diff
@@ -1,31 +1,40 @@
 import * as babel from '@babel/core';
 import { createFilter } from 'rollup-pluginutils';
-import createPreflightCheck from './preflightCheck.js';
-import helperPlugin from './helperPlugin.js';
-import { addBabelPlugin, escapeRegExpCharacters, warnOnce } from './utils.js';
-import { RUNTIME, EXTERNAL, HELPERS } from './constants.js';
+import preflightCheck from './preflightCheck.js';
+import bundledHelpersPlugin from './bundledHelpersPlugin.js';
+import { addBabelPlugin, escapeRegExpCharacters } from './utils.js';
+import { RUNTIME, EXTERNAL, BUNDLED, INLINE, HELPERS } from './constants.js';
 
 const unpackOptions = ({
 	extensions = babel.DEFAULT_EXTENSIONS,
+	skipBabelHelpersCheck = false,
 	// rollup uses sourcemap, babel uses sourceMaps
 	// just normalize them here so people don't have to worry about it
 	sourcemap = true,
 	sourcemaps = true,
 	sourceMap = true,
 	sourceMaps = true,
 	...rest
-} = {}) => ({
-	extensions,
-	plugins: [],
-	sourceMaps: sourcemap && sourcemaps && sourceMap && sourceMaps,
-	...rest,
-	caller: {
-		name: 'rollup-plugin-babel',
-		supportsStaticESM: true,
-		supportsDynamicImport: true,
-		...rest.caller,
-	},
-});
+} = {}) => {
+	if (!rest.babelHelpers) {
+		throw new Error(
+			'You have to specify how do you want to bundle/import "Babel helpers" (runtime functions inserted by Babel which are used by some transformations).\n\n' +
+				'Please pass `babelHelpers` option to the rollup-plugin-babel with one of the following values:\n' +
+				`  - "${RUNTIME}" - you should use it especially when building libraries with rollup. It has to be used in combination with \`@babel/plugin-transform-runtime\` and you should also specify \`@babel/runtime\` as dependency of your package (don't forget to tell rollup to treat it is your external dependency when bundling for cjs & esm formats).\n` +
```

Comment:
Would it be possible (or even advisable) to auto-add `@babel/plugin-transform-runtime` if not included?

Reply:
Installing dependencies is rather beyond the scope of this plugin, but maybe we could at least validate if it's installed properly.

Reply:
Ah yeah that's reasonable! I was more suggesting we could insert the plugin into the list, but just checking for validity is fine :+1: